BoldGrid/crio
# Add Mega Menu card to Pro Features page

Resolves #5


Testing Instructions:
1. Activate theme without Crio Pro installed.
2. Ensure the number '1' is shown under Crio > Pro Features.
3. View the Pro Features page, and ensure the first item shown is the 'Mega Menu' Card and that it has the 'New' banner.
4. Leave the page, or refresh, and ensure the 'New' banner disappears and that the '1' is no longer shown under Crio > Pro Features.
